src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpDebuggerIntellisenseDesktop.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpDebuggerIntellisenseDesktop : CSharpDebuggerIntellisenseCommon
```

Comment:
See comment in other file: xUnit lets you put facts in base types directly, and it's enumerated in all derived types.